kolibri/plugins/coach/assets/src/vue/coach-exercise-render-page/answer-history/index.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+
+  <div id="answer-history">
+    <h3>{{ $tr('header') }}</h3>
+
+    <ul id="history-list">
+      <template v-for="(attemptLog, index) in reformatedAttemptLogs">
+        <li v-if="index == 0">
```

Comment:
this should be either `index === 0` or `!index`

http://eslint.org/docs/rules/eqeqeq